test/unit/io_managers_test.py

```diff
@@ -0,0 +1,80 @@
+"""Test Dagster IO Managers."""
```

Comment:
Whee, tests! Looks like this only tests the `SQLiteIOManager` for now, and like we're not testing the view generation - should we test the other IOManagers here? I'm happy to merge first and push more tests as a follow-up too, since this PR is already pretty big.

Reply:
Ah yes, we should add tests for the other IOManagers and the view generation. We could create a new PR off of this branch.